.github/dependabot.yml

```diff
@@ -0,0 +1,53 @@
+version: 2
```

Comment:
Please remove .github directory, otherwise we get weird PRs.

Reply:
Done.